ci/matrix.yaml

```diff
@@ -11,6 +11,49 @@ workflows:
 
   pull_request:
     # Old CTK
+    - {jobs: ['build'], std: 'all', ctk: '11.1', cxx: ['gcc6', 'clang9', 'msvc2017']}
+    - {jobs: ['build'], std: 'all', ctk: '11.8', cxx: ['gcc11'], sm: '60;70;80;90'}
+    # Current CTK build-only
+    - {jobs: ['build'], std: 'all', cxx: ['gcc7', 'clang9', 'msvc2019']}
+    - {jobs: ['build'], std: 'all', cxx: ['gcc', 'clang', 'msvc', 'intel']}
```

Comment:
question: What does it mean when a compiler name like `gcc` is specified without a version?

Reply:
It's a shortcut to always use the latest version.